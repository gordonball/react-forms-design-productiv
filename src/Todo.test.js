import { render } from "@testing-library/react";
import Todo from "./Todo.js";

const todo = {
  id: 1,
  title: "Test",
  description: "Test",
  priority: 2
}

describe("Todo", function() {
  it("renders without crashing", function() {
    render(<Todo todo={todo} />);
  });

  it("matches snapshot", function() {
    const { container } = render(<Todo todo={todo} />);

    expect(container).toMatchSnapshot();
  });
});
