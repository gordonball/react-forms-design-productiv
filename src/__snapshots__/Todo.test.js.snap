// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        Test Title
      </b>
       
      <small>
        (priority: 
        2
        )
      </small>
    </div>
    <div>
      <small>
        Test Description
      </small>
    </div>
  </div>
</div>
`;
